modules/candidate_parameters/test/candidate_parametersTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * candidate_parameters automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class candidateParametersTestIntegrationTest extends LorisIntegrationTest
+{
+    /**
+     * Tests that, when loading the candidate_parameters module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testCandidateParametersDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=candidate_parameters");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Candidate Parameters", $bodyText);
```

Comment:
missing subtests: add_family, update_participant_status, update_candidate_info, update_proband_info, and update_consent_info